src/test/java/runner/RunCukesTest.java

```diff
@@ -30,7 +30,7 @@
                 "json:target/cucumber.json",
                 "rerun:target/rerun.txt"},
         glue = {"steps", "hooks"},
-        features = {"src/test/resources/features/"},
+        features = {"src/test/resources/features/Login.feature"},
```

Comment:
Please just point to "features" (as it was before), so you can run all your scenarios.

Reply:
done